src/utilities/link/types.ts

```diff
@@ -1,13 +1,13 @@
 export interface LinkLikeComponentProps
   extends React.HTMLProps<HTMLAnchorElement> {
-  /** A destination to link to */
+  /** The url to link to */
   url: string;
-  /** Forces url to open in a new tab */
+  /**	The content to display inside the link */
+  children?: React.ReactNode;
+  /** Makes the link open in a new tab */
   external?: boolean;
-  /** Tells the browser to download the url instead of opening it. Provides a hint for the downloaded filename if it is a string value. */
+  /** Makes the browser to download the url instead of opening it. Provides a hint for the downloaded filename if it is a string value. */
```

Comment:
```suggestion
  /** Makes the browser download the url instead of opening it. Provides a hint for the downloaded filename if it is a string value. */
```